packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -134,6 +136,10 @@ async function main(): Promise<number | void> {
     }
   }
 
+  if (args['--inspect-brk']) {
+    jestCli = jestCli.withDebugger()
+  }
```

Comment:
Unrelated changes, I can split them off to a separate PR.

Reply:
I've figured they were unrelated, but it's fine to keep them here.